### PR TITLE
cachelittle improvement ... this way run in my server in 0.2 sec

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
see code

### CODE REVIEW

File: tests/unit/suites/libraries/joomla/cache/storage/JCacheStorageCacheliteTest.php [hunk 1]
```diff
@@ -58,14 +58,30 @@ public function testCacheTimeout()
...
+		$interval = 0.1;
+
+		do
+		{
+			$timer += $interval;
+
+			usleep($interval * 1000000);
+
+			$cache = $this->handler->get($this->id, $this->group);
+
```
Comment (csthomas) [inline tests/unit/suites/libraries/joomla/cache/storage/JCacheStorageCacheliteTest.php:84]: How many loops is executed? I think that first loop should return valid data, second may return false.
Then we will be sure that cache works at all.

File: tests/unit/suites/libraries/joomla/cache/storage/JCacheStorageCacheliteTest.php [hunk 2]
```diff
@@ -58,14 +58,29 @@ public function testCacheTimeout()
 	{
 		/** @var Cache_Lite $cacheLiteInstance */
 		$cacheLiteInstance = TestReflection::getValue('JCacheStorageCachelite', 'CacheLiteInstance');
-		$cacheLiteInstance->_lifeTime = 2;
+		$cacheLiteInstance->_lifeTime = 0.1;
 
 		$data = 'testData';
 
 		$this->assertTrue($this->handler->store($this->id, $this->group, $data), 'Initial Store Failed');
```
Comment (csthomas) [inline tests/unit/suites/libraries/joomla/cache/storage/JCacheStorageCacheliteTest.php:66]: Put here something like:

```
// Test whether data was stored.
$cache  = $this->handler->get($this->id, $this->group);
$this->assertEquals($data, $cache, 'Some data should be available in lifetime.');
```

Comment (andrepereiradasilva) [reply tests/unit/suites/libraries/joomla/cache/storage/JCacheStorageCacheliteTest.php:66]: done in both

File: tests/unit/suites/libraries/joomla/cache/storage/JCacheStorageCacheliteTest.php (hunk 1 above)
Comment (andrepereiradasilva) [reply tests/unit/suites/libraries/joomla/cache/storage/JCacheStorageCacheliteTest.php:84]: normally 1 to 3 cycles
it works i tested!
put and `echo $cache;` here and you will see

**Pull request merged**